pilot/lib/pilot/build_manager.rb

```diff
@@ -121,7 +121,7 @@ def describe_build(build)
     end
 
     def should_update_build_information(options)
```

Comment:
If you have some extra minutes, maybe you can update the method name to follow the Ruby convention of ending with a question mark for methods that return boolean values => `should_update_build_information?` 👍